jsapp/js/account/accountFieldsEditor.module.scss

```diff
@@ -0,0 +1,9 @@
+@use 'scss/sizes';
+
+.row {
+  width: 100%;
```

Comment:
On the old account profile page, the Country and City lines were both half-width:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/da3d3c9f-44a5-4718-b1ff-90c119e96c53)

Not sure if that's an intentional change or not.

Reply:
Right, forgot to update it. I will make this follow the figma designs as close as possible (given the fact that not all fields displayed in Figma exist in this PR)

Reply:
So given these facts:
1. There is no Figma design for Account Settings
2. There is Figma design for Registration screen, which has a few more and few less fields than Account Settings
3. Some of the fields in Figma design for Registration screen don't exist yet in the code

I've changed the order of the fields and put some in the same row. I've also added `label` prop to `KoboSelect` and fixed some styles in it (could use some more tweaking and fixing, but I didn't want to go too far with it). And I've changed that `TextBox` L size to match the `Button` (again). It should fix most of the UI problems with the `TextBox` (that I've introduced with that big size)